apps/avatax/src/modules/taxes/tax-provider-webhook.ts

```diff
@@ -20,5 +20,5 @@ export interface ProviderWebhookService {
     payload: OrderConfirmedSubscriptionFragment,
     saleorOrder: SaleorOrder,
   ) => Promise<CreateOrderResponse>;
-  cancelOrder: (payload: OrderCancelledPayload) => Promise<void>;
+  cancelOrder: (avataxId: string) => Promise<void>;
```

Comment:
avoid using strings, previous `{avataxId: string}` is better, because you have more flexibility to modify code without huge refactor. 

also, it forces you to "See" what you enter into argument (named arguments).

When I see

`cancelOrder(...string type)` my first thought is to put there "order id"